modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheAbstractFailoverSelfTest.java

```diff
@@ -228,8 +228,9 @@ protected void testConstantTopologyChange(@Nullable final TransactionConcurrency
             }
         }, TOP_CHANGE_THREAD_CNT, "topology-change-thread");
 
+        boolean isInterrupted = false;
```

Comment:
Add empty line between variable declaration and try block, because they are different semantic units.

Reply:
Thanks a lot